test/tests/functional/pbs_ralter.py

```diff
@@ -1108,3 +1109,153 @@ def test_conflict_standing_resv_occurrence(self):
 
         self.alter_a_reservation(rid, start, end, shift, alter_e=True,
                                  whichMessage=0)
+
+    def test_alter_advance_resv_boundary_values(self):
+        """
+        This test checks the alter of start and end times at the boundary
+        values for advance reservation
+        """
```

Comment:
I'm not sure I understand what boundaries you are talking about here.  All this test does is move the end time, move it back, and do the same thing with the start time.  There are no boundaries that the alter is butting up against.

Reply:
The start and end time for the reservation are the input data for "pbs_ralter" feature here.
While designing the test cases using the equivalence partitioning approach, these start and end time become the boundary values around which the tests are designed. Hence mentioned the boundaries.